spec/lucky/content_type_helper_spec.cr

```diff
@@ -55,6 +55,20 @@ describe Lucky::ContentTypeHelpers do
       FakeAction.new.xml?.should be_false
     end
   end
+
+  it "works for plain text" do
+    set_content_type "text/plain" do
+      FakeAction.new.plain?.should be_true
+    end
+
+    set_content_type "text/plain; charset=UTF8" do
+      FakeAction.new.plain?.should be_true
+    end
+
+    set_content_type "plain/text" do
```

Comment:
Is this in the spec or used in the wild? I haven’t seen it before but if it’s a common thing then I think it makes sense to add. Just want to make sure

Reply:
No, I often catch myself typing that wrong, so I figured I'd use it. I guess it's sort of irrelevant because it could really be replaced with anything. 🤷‍♂️

Reply:
In that case, I think removing it would be best since it isn't technically correct